### PR TITLE
correct some line continuations

### LINKED ISSUE #43: proper line continuations
from #39 

"proper" line continuation style puts the operators at the start of the lines, so the operations are clearly visible. Not needed though
Example (corrected) from `studycog.py`:
```py
embed = embed.add_field(name='Answers:', value='**A:** ' + question['answers']['A']
                        + '\n**B:** ' + question['answers']['B']
                        + '\n**C:** ' + question['answers']['C']
                        + '\n**D:** ' + question['answers']['D'], 
                        inline=False)
```

### PR DESCRIPTION
**Pull request by classabbyamp**
Fixes #43

### CODE REVIEW

File: exts/lookup.py [hunk 1]
```diff
@@ -48,14 +48,10 @@ def __init__(self, bot):
...
                     data = self.cty[query]
-                    embed.add_field(name="Entity",
-                                    value=data['entity'])\
-                         .add_field(name="CQ Zone",
-                                    value=data['cq'])\
-                         .add_field(name="ITU Zone",
-                                    value=data['itu'])\
-                         .add_field(name="Continent",
-                                    value=data['continent'])\
+                    embed.add_field(name="Entity", value=data['entity'])\
```
Comment (0x5c) [inline exts/lookup.py]: The backslashes are improper line continuations (especially in this case)

File: exts/study.py [hunk 2]
```diff
@@ -78,8 +78,7 @@ def __init__(self, bot: commands.Bot):
             embed.add_field(name='Answers:', value='**A:** ' + question['answers']['A'] +
                             '\n**B:** ' + question['answers']['B'] +
```
Comment (0x5c) [inline exts/study.py]: The issue calls for the concatenation operator to be at the start the the line

File: exts/grid.py [hunk 3]
```diff
@@ -83,18 +83,18 @@ def __init__(self, bot: commands.Bot):
                     # Haversine formula
                     d_lat = math.radians(loc2[0] - loc[0])
                     d_lon = math.radians(loc2[1] - loc[1])
-                    a = math.sin(d_lat/2) ** 2 +\
-                        math.cos(math.radians(loc[0])) * math.cos(math.radians(loc2[0])) *\
-                        math.sin(d_lon/2) ** 2
+                    a = math.sin(d_lat/2) ** 2
+                    a += math.cos(math.radians(loc[0])) * math.cos(math.radians(loc2[0]))
+                    a *= math.sin(d_lon/2) ** 2
```
[the remark this answers is not in the document]
Comment (classabbyamp) [reply exts/grid.py]: OOP should be fine, I copied the formulas from wiki I think

[the remark this answers is not in the document]
Comment (0x5c) [reply exts/grid.py]: Good, if OOP is fine, I'd put each operand on a new line (breaking up line 2)

File: exts/grid.py [hunk 4]
```diff
@@ -83,18 +83,18 @@ def __init__(self, bot: commands.Bot):
...
                     d_mi = 0.6213712 * d
 
                     # Bearing
                     y_dist = math.sin(math.radians(loc2[1]-loc[1])) * math.cos(math.radians(loc2[0]))
-                    x_dist = math.cos(math.radians(loc[0])) * math.sin(math.radians(loc2[0])) -\
-                        math.sin(math.radians(loc[0])) * math.cos(math.radians(loc2[0])) *\
-                        math.cos(math.radians(loc2[1] - loc[1]))
+                    x_dist = math.cos(math.radians(loc[0])) * math.sin(math.radians(loc2[0]))
+                    x_dist -= math.sin(math.radians(loc[0])) * math.cos(math.radians(loc2[0]))
+                    x_dist *= math.cos(math.radians(loc2[1] - loc[1]))
```
[the remark this answers is not in the document]
Comment (0x5c) [reply exts/grid.py]: > If OOP is fine, I'd put each operand on a new line (breaking up line 2)

Same here

[the remark this answers is not in the document]
Comment (classabbyamp) [reply exts/grid.py]: done

**Pull request merged**